ERCS/erc-7786.md

```diff
@@ -109,6 +116,14 @@ After a sender has invoked `sendMessage`, further action MAY be required by the
 
 The exact interface for any such action is out of scope of this ERC. If the `postProcessingOwner` attribute is supported and present, such actions MUST be restricted to the specified account, otherwise they MUST be able to be performed by any party in a way that MUST NOT be able to compromise the eventual receipt of the message.
 
+#### `requestRelay`
+
+Requests that the message identified by `outboxId` be delivered to the receiver (i.e., relayed).
+
+Pre-pays via `msg.value` for the execution of the message on the receiver up to `gasLimit`. A `relayerFee` should be deducted from this to compensate the relayer.
```

Comment:
there may be additional fees besides just the relayer fee. Such as, do any protocols just charge protocol fees at the gateway itself? I feel like it might be better to let all of the fees be sort of opaque, instead of specifying the fees separately from the gas needed for execution. I'd argue the user doesn't really care about the breakdown; just really care how much they need to pay to get the message executed at the destination.

Reply:
Replying to this in https://github.com/ethereum/ERCs/pull/981#discussion_r2004117940.